Workshop organizing guide (#489)
precice/precice.github.io: pages/community/precice-workshop-organization.md

- Fitting around 50 people (but not much larger than that)
- For the hands-on training course and the user support sessions: easy access at every seat, (collaborative) working surface, enough power sockets, stable internet connection
- For the talks: good acoustics and visibility, recording infrastructure
- For the world café: (standing) tables, placed far from each other, isolated from noisy areas of the building
- For the catering: ideally next to the rooms, isolated, standing

With "recording infrastructure", we mean infrastructure to capture the speaker audio+video, slides, and an additional microphone in separate stream (a meeting camera system is typically not fitting the purpose). We can provide specialized hardware to do the mixing of these channels into what you later see as recordings on YouTube, if this is not already available in the room.

The rooms for the training and the talks are typically different, even if the schedule is typically structured as a single track. The catering and the world café can typically use the same room and tables, but reusing the collaborative working room for the world café is also fine.

### Define financing scheme

> Typically, the workshop is funded by participation fees. An important dependency for subsequent steps is which entity can collect and process such participation fees. This might be an association supporting scientific activities, an event management unit of a university, or the organizing institute (e.g., via project infrastructure).

Reviewer: ```suggestion
Typically, the workshop is co-funded by participation fees. An important dependency for subsequent steps is which entity can collect and process such participation fees. This might be an association supporting scientific activities, an event management unit of a university, or an external company. The setup should allow to model an income to reinvest into maintaining preCICE.
```
How is "event management unit of a university" different from "organizing institute"?

Author: I meant that there might be a dedicated service at the university that supports organizing events, instead of the chair itself organizing this by themselves without institutional support.

But this is a better formulation.

Multiple types of tickets are typically defined:

- early -vs- late registration
- full price -vs- discount for academia
- training course included -vs- only main part of the workshop

In addition, free registration is accounted for:

- holders of a support license
- invited speakers

> - special cases

Reviewer: ```suggestion
- organizers (if possible)
- special cases
```

### Set milestones

Set clear responsible persons for each task, and track progress in a collaborative project board.

Important dates to set:

- "save the date" announcement: one year ahead
- call for contributions (needs a form): open six months ahead. An early call mainly acts as advertisement.

> - open registration (needs a form): six months ahead. Early registration is important for participants that need a visa application to travel.

Reviewer: Sounds too early to me. Of course, early does not hurt. Visa application should only depend on the contribution, not on the registration, no?

> - abstract submission deadline and call acceptance: the earlier the better. Can be after the early-bird registration deadline. The call acceptance can be as short as one week after the deadline.

Reviewer: Why after the early-bird registration deadline? Should be before, no?

- early-bird registration deadline: important for estimating the number of participants, driven by the catering requirements.

> - request catering offers: typically 1-2 months before the workshop.

Reviewer: sounds too late to me

Author: ```suggestion
- request catering offers: typically 3-4 months before the workshop.
```

> - reserve tables for the workshop dinner: typically 1-2 months before the workshop.

Reviewer: sounds too late to me

Author: ```suggestion
- reserve tables for the workshop dinner: typically 3-4 months before the workshop.
```

- late registration deadline: typically one week before the workshop, does not affect much.

## Registration infrastructure

Two forms are mainly needed, and it is easier if they are decoupled: an abstract submission form, and a registration form.

The abstract submission form should collect:

- Kind of contribution (talk or poster, if offered)

> - Presenter

Reviewer: ```suggestion
- Presenter
- Authors
```

Author: I am not sure if we have collected the list of authors in the past.
On the one hand, it would help us identify where the contribution comes from.
On the other hand, it might be an obstacle to people that first need to discuss a seemingly final authors list (depends on the local culture).

I would suggest having an explicit note next to it:

```suggestion
- Presenter
- Authors (with the note: "preliminary list, can be modified later")
```

- Affiliation
- Contact email
- Title

> - Abstract (plain text, short)

Reviewer: ```suggestion
- Abstract (plain text, max 1500 chars including spaces)
```

Author: This sounds too specific and arbitrary to me for a future-proof guide, but I guess we can use it as a guideline of what we mean "short".

- A disclaimer that, if accepted, this information is going to appear on the website

This form can be implemented in any survey tool. Live access to the data by the preCICE team would be helpful, to understand where further advertising might be needed.

The registration form should collect:

> - Name
> - Affiliation

Reviewer: ```suggestion
- Name
- Affiliation
- Billing address (depending on the financial setup)
```

- Contact email
- Dates/modules that one plans to attend:
  - Training course
  - Main workshop
  - Dinner
  - User support session (if the only event of the day)
  - Potential other social events

> - Permission to include contact details in a participants list distributed to the participants

Reviewer: ```suggestion
- Dietary restrictions
- Permission to include contact details in a participants list distributed to the participants
```

- A disclaimer that photos are going to be taken on site and will be used for advertising
- Potential questions of interest for the workshop (particular topics of interest, goals)

> This form is typically implemented by the entity processing the payments, which also issues invoices, receipts. Who issues confirmation of participation depends on the local infrastructure available. If not automatic, the organizers can provide a letter at the end to those requesting it.

Reviewer: ```suggestion
This form is typically implemented by the local organizers. Payments including invoices or receipts could be part of the process or separate.
```
So far, INFOS was the only entity that directly included payments.

With confirmation, you mean letter of attendance here, right? Could be confusing like this.

> After submitting either form, the applicant should get an automatic confirmation (feature not available in some tools, e.g., Nextcloud).

Reviewer: ```suggestion
After submitting either form, the applicant should get an automatic confirmation (feature not available in some tools, e.g., Nextcloud), which includes information on how to pay the registration fee.
```
